### PR TITLE
Add tutorial initiation trigger to start command, add tutorial to help text, add utterance identification function, update tutorial middleware to always identify messages.

### PR DESCRIPTION
**Pull request by arthurshir**
## Changes
* Add tutorial initiation trigger to start command
* Add tutorial to help text
* Separate out start command from help command in bot
* Implement `utteranceHasForwardSlashCommand` in botfunctions.utils
* Fix typo in tutorial initiation
* Add start_command event to tutorial FSM
* Add start_command event identification
* update unit tests
* update tutorial middleware to always attempt to identify messages.
* update tutorial event identifier logging to only log when a tutorial event occurs.

## Testing Done
* Tested locally
* Updated unit tests

### CODE REVIEW

File: tutorial/eventManager.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ let fsm = new FSM(transitions, STATES.NOT_IN_TUTORIAL);
 
 const REPLY_PARAMS = {
     TUTORIAL_INITIATION: {
-        TEXT: "❕Hello there, lets walk you through what you can do in the bot, it takes less than a minute!",
+        TEXT: "❕Lets walk you through what you can do in the bot, it takes less than a minute!",
```
Comment (bubakazouba) [inline tutorial/eventManager.js:35]: Let's*

File: botfunctions/helpcommand.js [hunk 2]
```diff
@@ -67,8 +69,10 @@ function addToBot(bot) {
 
     bot.command('start', ctx => {
         logger.info(`${ctx.chat.id}: COMMAND_START`);
-        UserManager.addUser(ctx.chat.id, ctx.chat.username);
-        return ctx.replyWithHTML('Hi there 👋! This is a simple bot that helps you remember things.' + '\n' + HELP_TEXT_ENGLISH).catch(UserManager.catchBlocks);
+        let userId = ctx.chat.id;
+        UserManager.addUser(userId, ctx.chat.username);
+        ctx.replyWithHTML('Hi there 👋! This is a simple bot that helps you remember things.').catch(UserManager.catchBlocks);
+        return tutorial.registerAndReactToAskToStartTutorialEvent(userId, ctx);
```
Comment (bubakazouba) [inline botfunctions/helpcommand.js:75]: this can be handled nicely in the FSM
event = /start -> start_tutorial?

Comment (bubakazouba) [reply botfunctions/helpcommand.js:75]: the whole point of the `tutorial` module, the middleware ..etc is to centralize all of the tutorial stuff in one place, let's stick with the pattern

Comment (arthurshir) [reply botfunctions/helpcommand.js:75]: true... ok will do

File: botfunctions/startcommand.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    constants = require("./constants.js");
+
+const COMMAND_REGEX = utils.makeRegexForCommandStrings(constants.START_COMMAND_VARIANTS);
+
+function addToBot(bot) {
+    bot.command('start', ctx => {
+        logger.info(`${ctx.chat.id}: COMMAND_START`);
+        let userId = ctx.chat.id;
+        UserManager.addUser(userId, ctx.chat.username);
+        return ctx.replyWithHTML('Hi there 👋! This is a simple bot that helps you remember things.').catch(UserManager.catchBlocks);
```
Comment (bubakazouba) [inline botfunctions/startcommand.js:13]: add help text heree

**Comment by arthurshir** [comment]
Splitting this into multiple PRs

**Pull request closed without merging**